client/app/index.jsx

```diff
@@ -1,4 +1,5 @@
 import './lib/polyfills'
+import 'babel-polyfill'
```

Comment:
Oh right, I forgot to check that and to remove these polyfills. Good catch, thanks ;)